### ISSUE TITLE
Docker checksum fails - solution inside

### ISSUE THREAD

**Issue by obarlev** [SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D#33]
Hi,
Docker checksum fails while pulling most images I tested (private and public registries).

```filesystem layer verification failed for digest sha256: ```

After a lot of digging I've found this issue:
https://github.com/docker/for-mac/issues/4654

The offered solution is to remove the values of "Cpuid1Data" and "Cpuid1Mask".
Per my tests it works, just wondering what that means to keep those fields with no values? any expected impact?

**Comment by morningex** [comment]
Hi @obarlev, thanks for your guide. I am experiencing the same issue. Do you mean setting `Cpuid1Data` and `Cpuid1Mask`in  config.plist to all zero? or you mean remove entire variable from config.plist

**Comment by obarlev** [comment]
@morningex exactly, just leave them blank (no zero needed as far as I know). 
I made this change a week ago and haven't experienced any issue so far.

**Comment by SchmockLord** [comment]
Yeah, since 10.5.5 you don't need this Fake-CPU-ID anymore. But I some people already told me, that they needed to add it again for the Big Sur Beta. So I just left it in my config as long as this produces no issues.

**Comment by klwlau** [comment]
I am runnin 10.15.6. Removing Cpuid1Data and Cpuid1Mask makes my system fail to boot.

**Comment by sealingtiper** [comment]
> I am runnin 10.15.6. On intel nuc 10 i7 (10710U). Removing Cpuid1Data and Cpuid1Mask makes my system fail to boot.

Try this https://www.tonymacx86.com/threads/intel-nuc-10-frost-canyon.289485/page-35#post-2153839

**Comment by klwlau** [comment]
> > I am runnin 10.15.6. On intel nuc 10 i7 (10710U). Removing Cpuid1Data and Cpuid1Mask makes my system fail to boot.
> 
> Try this https://www.tonymacx86.com/threads/intel-nuc-10-frost-canyon.289485/page-35#post-2153839

this is not working for me. It can pass all the loading and debug information (the white text showing before booting into macOS). After loading all the white text, it stuck at black screen forever.

**Comment by sealingtiper** [comment]
> > > I am runnin 10.15.6. On intel nuc 10 i7 (10710U). Removing Cpuid1Data and Cpuid1Mask makes my system fail to boot.
> > 
> > 
> > Try this https://www.tonymacx86.com/threads/intel-nuc-10-frost-canyon.289485/page-35#post-2153839
> 
> this is not working for me. It can pass all the loading and debug information (the white text showing before booting into macOS). After loading all the white text, it stuck at black screen forever.

Sorry about that.
For me, I have NUC10i7FNB, using the EFI at 362 floor (https://www.tonymacx86.com/threads/intel-nuc-10-frost-canyon.289485/post-2155267),
 It boot successfully. Everything works fine except docker.  when i was pulling images , always got a `filesystem layer verification failed for digest sha256:` error .
Then I tried the method to change value like
Data: <XX XX XX XX 00 00 00 00 00 00 00 08 00 00 00 00>
Mask: <FF FF FF FF 00 00 00 00 00 00 00 08 00 00 00 00>
It fix my problem.
I'm a newbie for hackintosh, sorry for can not give more ideas.

**Comment by klwlau** [comment]
> i

appreciate your help
i hv the same error too
after applying your suggestion, my system cannot boot
which version of OS are you running on? i am at 10.15.6

**Comment by sealingtiper** [comment]
> > i
> 
> appreciate your help
> i hv the same error too
> after applying your suggestion, my system cannot boot
> which version of OS are you running on? i am at 10.15.6

I installed 10.15.6 too.
Note that XXXX in the Data section is your origin data, not x. and the '08' is change from '00'

**Comment by klwlau** [comment]
yes, I did what the above suggest. can you share what xxxx and cpu model are you using?

**Comment by klwlau** [comment]
> > > i
> > 
> > 
> > appreciate your help
> > i hv the same error too
> > after applying your suggestion, my system cannot boot
> > which version of OS are you running on? i am at 10.15.6
> 
> I installed 10.15.6 too.
> Note that XXXX in the Data section is your origin data, not x. and the '08' is change from '00'

yes, I did what the above suggest. can you share what xxxx and cpu model are you using?

**Comment by sealingtiper** [comment]
> > > > i
> > > 
> > > 
> > > appreciate your help
> > > i hv the same error too
> > > after applying your suggestion, my system cannot boot
> > > which version of OS are you running on? i am at 10.15.6
> > 
> > 
> > I installed 10.15.6 too.
> > Note that XXXX in the Data section is your origin data, not x. and the '08' is change from '00'
> 
> yes, I did what the above suggest. can you share what xxxx and cpu model are you using?

Sure! 
<img width="676" alt="CPU" src="https://user-images.githubusercontent.com/18738957/89141406-2530c880-d577-11ea-878f-4acedb088242.png">

**Comment by klwlau** [comment]
> > > > > i
> > > > 
> > > > 
> > > > appreciate your help
> > > > i hv the same error too
> > > > after applying your suggestion, my system cannot boot
> > > > which version of OS are you running on? i am at 10.15.6
> > > 
> > > 
> > > I installed 10.15.6 too.
> > > Note that XXXX in the Data section is your origin data, not x. and the '08' is change from '00'
> > 
> > 
> > yes, I did what the above suggest. can you share what xxxx and cpu model are you using?
> 
> Sure!
> <img alt="CPU" width="676" src="https://user-images.githubusercontent.com/18738957/89141406-2530c880-d577-11ea-878f-4acedb088242.png">

I am using the exact value running on a i7-10710U

**Comment by sealingtiper** [comment]
> > > > > > i
> > > > > 
> > > > > 
> > > > > appreciate your help
> > > > > i hv the same error too
> > > > > after applying your suggestion, my system cannot boot
> > > > > which version of OS are you running on? i am at 10.15.6
> > > > 
> > > > 
> > > > I installed 10.15.6 too.
> > > > Note that XXXX in the Data section is your origin data, not x. and the '08' is change from '00'
> > > 
> > > 
> > > yes, I did what the above suggest. can you share what xxxx and cpu model are you using?
> > 
> > 
> > Sure!
> > <img alt="CPU" width="676" src="https://user-images.githubusercontent.com/18738957/89141406-2530c880-d577-11ea-878f-4acedb088242.png">
> 
> I am using the exact value running on a i7-10710U. But I cannot boot

Sorry for that, and I'm totally a newbie in hackintosh.Maybe you can try to ask for some help at www.tonymacx86.com. People  are kindly there.

**Comment by klwlau** [comment]
> > > > > > > i
> > > > > > 
> > > > > > 
> > > > > > appreciate your help
> > > > > > i hv the same error too
> > > > > > after applying your suggestion, my system cannot boot
> > > > > > which version of OS are you running on? i am at 10.15.6
> > > > > 
> > > > > 
> > > > > I installed 10.15.6 too.
> > > > > Note that XXXX in the Data section is your origin data, not x. and the '08' is change from '00'
> > > > 
> > > > 
> > > > yes, I did what the above suggest. can you share what xxxx and cpu model are you using?
> > > 
> > > 
> > > Sure!
> > > <img alt="CPU" width="676" src="https://user-images.githubusercontent.com/18738957/89141406-2530c880-d577-11ea-878f-4acedb088242.png">
> > 
> > 
> > I am using the exact value running on a i7-10710U. But I cannot boot
> 
> Sorry for that, and I'm totally a newbie in hackintosh.Maybe you can try to ask for some help at [www.tonymacx86.com](http://www.tonymacx86.com). People are kindly there.

no worries, I am new to it too. Thanks for your help appreciated

**Comment by bugsplat490** [comment]
With the latest OpenCore 0.6.1 files in the master branch of this repo at the time of this comment, it looks like this issue in [acidanthera](https://github.com/acidanthera/bugtracker/issues/1035) has now been merged in and appears in this repo. I can not reproduce this issue and likely this bug should now be closed...

**Comment by jkcom** [comment]
@klwlau Did you find a solution to this issue? I have exact same NUC 10 i7 and running into same problem.

**Comment by morningex** [comment]
@jkcom yes i do.

1. Follow the above modification on plist.
2. Then reboot. The first rebook will lead to a black freeze screen. It is normal. 
3. After you reach the freeze, press and hold the power button till it force shut down. 
4. Unplug the physical power connector at the end of the Nuc 10
5. Wait 5 second
6. Plug it back and start the machine

**Comment by jkcom** [comment]
@morningex Thank you so much! I only tried with empty values, but using the edited and the reboot did the trick!
